cross/shairport/Makefile

```diff
@@ -0,0 +1,34 @@
+PKG_NAME = shairport
+PKG_VERS = 2.2.3
+PKG_EXT = zip
+PKG_DIST_NAME = $(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/mikebrady/shairport-sync/archive
+PKG_DIR = $(PKG_NAME)-sync-$(PKG_VERS)
+
+DEPENDS = cross/libtool cross/popt cross/libdaemon cross/openssl cross/alsa-lib
+HOMEPAGE = https://github.com/mikebrady/shairport-sync
+COMMENT  = Use Synology NAS as an Airplay receiver. Requires USB soundcard or speakers.
+LICENSE  =
+
+FINAL_INSTALL_DIR  = /usr/local/$(PKG_NAME)
+FINAL_VAR_DIR = $(FINAL_INSTALL_DIR)/var
+
+GNU_CONFIGURE = 1
+
+CONFIGURE_ARGS = --prefix=$(STAGING_INSTALL_PREFIX) --with-alsa --with-ssl=openssl --with-tinysvcmdns --with-piddir=$(FINAL_VAR_DIR)
```

Comment:
`$(INSTALL_PREFIX)/var` should work here iirc. Then you can drop the two vars on L13,14.